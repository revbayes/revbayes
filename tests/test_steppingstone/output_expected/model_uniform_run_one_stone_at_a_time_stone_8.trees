Iteration	Posterior	Likelihood	Prior	psi
0	-26650.52	-26649.18	-1.345983	((((Aotus_trivirgatus[&index=1]:0.129484,(Cebus_albifrons[&index=3]:0.047470,Saimiri_sciureus[&index=20]:0.248912)[&index=33]:0.181427)[&index=41]:0.023887,Loris_tardigradus[&index=12]:0.099445)[&index=26]:0.011277,((Cheirogaleus_major[&index=4]:0.093527,(Lemur_catta[&index=10]:0.237878,Colobus_guereza[&index=6]:0.049626)[&index=43]:0.055358)[&index=31]:0.069564,((((((Macaca_mulatta[&index=13]:0.186458,Galago_senegalensis[&index=8]:0.120119)[&index=28]:0.065869,Perodicticus_potto[&index=18]:0.279718)[&index=34]:0.135193,Chlorocebus_aethiops[&index=5]:0.065832)[&index=32]:0.016717,(Microcebus_murinus[&index=14]:0.058772,(Tarsius_syrichta[&index=21]:0.162155,(Pan_paniscus[&index=17]:0.041558,(Daubentonia_madagascariensis[&index=7]:0.358445,Callicebus_donacophilus[&index=2]:0.011403)[&index=36]:0.279345)[&index=37]:0.279950)[&index=39]:0.018401)[&index=38]:0.019901)[&index=42]:0.053246,(Otolemur_crassicaudatus[&index=16]:0.153965,Propithecus_coquereli[&index=19]:0.162017)[&index=24]:0.020078)[&index=29]:0.048761,Varecia_variegata_variegata[&index=22]:0.123703)[&index=40]:0.032914)[&index=30]:0.106201)[&index=35]:0.191747,Galeopterus_variegatus[&index=23]:0.168022,(Nycticebus_coucang[&index=15]:0.062773,(Hylobates_lar[&index=9]:0.061361,Lepilemur_hubbardorum[&index=11]:0.090780)[&index=27]:0.078862)[&index=25]:0.097004)[&index=44];
10	-30250.31	-30243.51	-6.793917	(Cheirogaleus_major[&index=4]:0.152032,((((((Chlorocebus_aethiops[&index=5]:0.049701,((Hylobates_lar[&index=9]:0.079968,(Varecia_variegata_variegata[&index=22]:0.257125,((Otolemur_crassicaudatus[&index=16]:0.273519,Lepilemur_hubbardorum[&index=11]:0.108440)[&index=25]:0.160999,((Lemur_catta[&index=10]:0.164185,Loris_tardigradus[&index=12]:0.048163)[&index=40]:0.052820,((Galago_senegalensis[&index=8]:0.020290,(Tarsius_syrichta[&index=21]:0.146274,(((Callicebus_donacophilus[&index=2]:0.008667,Pan_paniscus[&index=17]:0.006681)[&index=38]:0.033091,Daubentonia_madagascariensis[&index=7]:0.323542)[&index=36]:0.205358,Propithecus_coquereli[&index=19]:0.080661)[&index=32]:0.079438)[&index=43]:0.185504)[&index=34]:0.098220,Colobus_guereza[&index=6]:0.043880)[&index=39]:0.057702)[&index=29]:0.122563)[&index=24]:0.009428)[&index=37]:0.381628)[&index=27]:0.077549,Nycticebus_coucang[&index=15]:0.033897)[&index=31]:0.041385)[&index=41]:0.022723,Microcebus_murinus[&index=14]:0.108273)[&index=42]:0.049812,Macaca_mulatta[&index=13]:0.210761)[&index=28]:0.035630,Perodicticus_potto[&index=18]:0.477595)[&index=35]:0.048824,Aotus_trivirgatus[&index=1]:0.268998)[&index=26]:0.021060,(Cebus_albifrons[&index=3]:0.030212,Saimiri_sciureus[&index=20]:0.059869)[&index=33]:0.202577)[&index=30]:0.057440,Galeopterus_variegatus[&index=23]:0.456174)[&index=44];
