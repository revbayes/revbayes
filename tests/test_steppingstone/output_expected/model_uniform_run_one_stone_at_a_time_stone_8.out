state	power	likelihood
4	0.00032	-29490
5	0.00032	-29036.7
6	0.00032	-28736.3
7	0.00032	-28491.6
8	0.00032	-28519.2
9	0.00032	-28624.3
10	0.00032	-28387.7
